editor/plugins/material_editor_plugin.h

```diff
@@ -71,27 +71,22 @@ class MaterialEditor : public Control {
 
 	HBoxContainer *layout_3d = nullptr;
 
-	TextureButton *sphere_switch = nullptr;
-	TextureButton *box_switch = nullptr;
-
-	TextureButton *light_1_switch = nullptr;
-	TextureButton *light_2_switch = nullptr;
-
 	Ref<Material> material;
 
+	Button *sphere_switch = nullptr;
+	Button *box_switch = nullptr;
+	Button *light_1_switch = nullptr;
+	Button *light_2_switch = nullptr;
+
 	struct ThemeCache {
-		Ref<Texture2D> light_1_on;
-		Ref<Texture2D> light_1_off;
-		Ref<Texture2D> light_2_on;
-		Ref<Texture2D> light_2_off;
-		Ref<Texture2D> sphere_on;
-		Ref<Texture2D> sphere_off;
-		Ref<Texture2D> box_on;
-		Ref<Texture2D> box_off;
+		Ref<Texture2D> light_1;
+		Ref<Texture2D> light_2;
+		Ref<Texture2D> sphere;
+		Ref<Texture2D> box;
```

Comment:
Since you're renaming them anyway, it would be good to suffix all 4 with `_icon` for clarity.